src/main/java/com/rackspacecloud/metrics/rollup/domain/Metric.java

```diff
@@ -1,118 +1,151 @@
 package com.rackspacecloud.metrics.rollup.domain;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
+@Data
 public class Metric implements IReducer<Metric> {
-    public String rollupKey;
 
-    public Map<String, List<Long>> iValuesForRollup;
-    public Map<String, List<Double>> fValuesForRollup;
+    private static final Logger LOGGER = LoggerFactory.getLogger(Metric.class);
```

Comment:
Since you're using lombok, you can use `@Slf4j` annotation on the class to inject the logger as `log`.